src/qml/components/ConnectionSettings.qml

```diff
@@ -45,15 +45,15 @@ ColumnLayout {
         header: qsTr("Networks")
         description: qsTr("Which networks to use for communication")
         actionItem: ValueInput {
-            description: qsTr("6")
+            description: "6"
         }
     }
     Setting {
         last: true
         Layout.fillWidth: true
         header: qsTr("Proxy settings")
         actionItem: ValueInput {
-            description: qsTr(">")
+            description: ">"
```

Comment:
we're supposed to use the caret-right icon now ... #144

Reply:
Addressed in 0b689ad874f9c4648d817657d948bb6ff03c79f5